src/components/TransactionsChartPanel.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <app-panel>
+    <template #heading>
+      TOTAL TRANSACTIONS
+    </template>
+    <template #header>
+      <transactions-chart-controls
+        class="transactions-chart-panel__controls"
+        @selected="fetchRangeData"/>
+    </template>
+
+    <div class="transactions-chart-panel__container">
+      <Line
+        :options="chartOptions"
+        :data="chartData"/>
+    </div>
+
+    <transactions-chart-controls
+      class="transactions-chart-panel__controls--condensed"
+      @selected="fetchRangeData"/>
+  </app-panel>
+</template>
+
+<script setup>
+import { Line } from 'vue-chartjs'
+import {
+  CategoryScale,
+  Chart as ChartJS,
+  Legend,
+  LinearScale,
+  LineElement,
+  PointElement,
+  Title,
+  Tooltip,
+} from 'chart.js'
+import { storeToRefs } from 'pinia'
+import { DateTime } from 'luxon'
+import { useTransactionsStore } from '~/stores/transactions'
+import TransactionsChartControls from '~/components/TransactionsChartControls'
+
+const transactionsStore = useTransactionsStore()
+const {
+  transactionsStatistics,
+} = storeToRefs(transactionsStore)
+const { fetchTransactionsStatistics } = transactionsStore
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+)
+
+ChartJS.defaults.font.family = 'Roboto Mono'
+
+const chartData = computed(() => {
+  return {
+    labels: labels.value,
+    datasets: [{
+      data: stats.value,
+      label: null,
+      cubicInterpolationMode: 'monotone',
+      tension: 0.4,
+      borderColor: '#f5274e',
+      backgroundColor: '#f5274e',
+      pointRadius: 3,
+      pointHitRadius: 20,
+    }],
+  }
+})
+
+const chartOptions = {
+  responsive: true,
+  maintainAspectRatio: false,
+  plugins: {
+    legend: {
+      display: false,
+    },
+    tooltip: {
+      tooltip: {
+        position: 'top',
+      },
+      callbacks: {
+        title: function(context) {
+          return context.label
+        },
+      },
+    },
+  },
+  scales: {
+    y: {
+      border: {
+        display: false,
+      },
+    },
+    x: {
+      grid: {
+        color: function() {
+          return 'transparent'
+        },
+      },
+    },
+  },
+}
+
+const selectedInterval = ref('')
+
+await fetchTransactionsStatistics()
```

Comment:
I think we should fetch it through useAsyncData to avoid fetching it twice - on SSR and second time on CSR

Reply:
fixed